statsqsmessageconsumer.go

```diff
@@ -109,9 +109,9 @@ func (t StatMessageConsumer) ConsumeMessage(ctx context.Context, message *sqs.Me
 	stat.Count(t.ConsumedSize, float64(len([]byte(*message.Body))))
 
 	var start = time.Now()
-	err = t.wrapped.ConsumeMessage(ctx, message)
+	consumeerr := t.wrapped.ConsumeMessage(ctx, message)
```

Comment:
Nit: can we change the variable name to `consumeErr`? `consumeerr` looks funny

Reply:
will do